code/modules/research/designs/equipment_designs.dm

```diff
@@ -189,4 +189,85 @@
 	build_type = PROTOLATHE
 	materials = list("$metal" = 200, "$glass" = 200, "$uranium" = 1000, "$plasma" = 200)
 	build_path = /obj/item/clothing/glasses/hud/hydroponic/night
+	category = list("Equipment")
+	
+
+/datum/design/handdrill
+	name = "Hand Drill"
+	desc = "A small electric hand drill with an interchangable screwdriver and bolt bit"
+	id = "handdrill"
+	req_tech = list("materials" = 4, "engineering" = 6)
+	build_type = PROTOLATHE
+	materials = list(MAT_METAL = 3500, MAT_SILVER = 1500, MAT_TITANIUM = 2500)
```

Comment:
How is MAT_TITANIUM even defined though? Strange. Will need to figure out an acceptable alternative to titanium.